avocado/utils/linux_modules.py

```diff
@@ -16,6 +16,8 @@
 #
 # Copyright: 2016 IBM
 # Authors : Praveen K Pandey <praveen@linux.vnet.ibm.com>
+# Copyright: 2017 IBM
+# Authors: Narasimhan V <sim@linux.vnet.ibm.com>
```

Comment:
Do you find this change so significant that you need to update the copyright? (the idea of `copyright/authors` is to mark authors who did a major update of that part and usually is used only to mention the first author(s). It can be updated occasionally for example when the module is completely reworked or extended in a such extend that the original work is not that significant anymore.)